lib/enum/arrays.rb

```diff
@@ -1,24 +1,86 @@
 class Array
+  def iniinitialize
```

Comment:
Careful there - you've changed the name of the initialize method, so now it won't run. That's why you had to use `self` in the method definitions. Now, using `self` is perfectly fine, but the initialize method exists so that you didn't have to do that and could use `@array` instead.